internal/pkg/template/templates/addons/s3/env/access_policy.yml

```diff
@@ -11,9 +11,7 @@ Parameters:
 Resources:
   {{logicalIDSafe .Name}}BucketAccessPolicy:
     Metadata:
-      'aws:copilot:description': !Sub
```

Comment:
Is this the only template with !Sub? in the metadata description

Reply:
actually no